pages/opam-using.html

```diff
@@ -6,25 +6,31 @@ <h2>What is opam?</h2>
 
 <p>Opam is the package manager for the OCaml programming language, the language
 in which Coq is implemented.
-Opam 2 is the recommended version, and is assumed below.
+Opam 2.1 is the recommended version, and is assumed below.
 Instructions on
 <a href="https://opam.ocaml.org/doc/Install.html">how to install opam</a>
 itself are available on the opam website.
 The following command displays the version of opam you have installed:</p>
 
-<pre><code># Make sure opam version is 2.0.0 or above.
+<pre><code># Make sure opam version is 2.1.0 or above.
 opam --version
 </code></pre>
 
 <p>Follow the instructions below to install the last stable version of Coq and
 additional packages. The instructions target an opam newcomer.</p>
 
+<p>Note that these instructions will also work for Opam 2.0 but this may require you to manually install any external dependencies. Opam will inform you what these are if they are required.</p>
```

Comment:
Please wrap your lines to make them easier to review.

Opam 2.0 does not inform precisely on which external dependencies are needed. This note should recommend running `opam-depext` to get the information on external dependencies when running Opam 2.0.